core/modules/file.py

```diff
@@ -656,7 +655,7 @@ def add(self, skelType: SkelType, node=None, *args, **kwargs):
         if skelType == "leaf":  # We need to handle leafs separately here
             skey = kwargs.get("skey")
```

Comment:
If `kwargs` doesn't contains the `skey` it's `None`. But the type hint enforces a `str` (`def validate(key: str, ...)`).

Therefore I suggest this change:

```suggestion
            skey = kwargs.get("skey", "")
```